source/includes/fact-no-mongosync-disaster-recovery.rst

```diff
@@ -0,0 +1,4 @@
+Until you've called commit on ``mongosync`` and ``canWrite`` successfully 
```

Comment:
can we link to https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/commit/#std-label-c2c-api-commit for commit